package.json

```diff
@@ -24,6 +24,7 @@
     "cors": "^2.8.5",
     "express": "^4.21.1",
     "morgan": "^1.10.0",
+    "sqlite": "^5.1.1",
```

Comment:
The slightly nicer sqlite package, which calls itself `node-sqlite` and is listed as `sqlite` on npm. We still need `sqlite3` which is used as the db driver.